src/BlazorWebView/src/Maui/Android/BlazorWebViewHandler.Android.cs

```diff
@@ -59,19 +59,37 @@ protected override AWebView CreatePlatformView()
 			return blazorAndroidWebView;
 		}
 
+		private const string AndroidFireAndForgetAsyncSwitch = "BlazorWebView.AndroidFireAndForgetAsync";
```

Comment:
Not to bikeshed, but would `"BlazorWebView.AndroidFireAndForgetDisposeAsync"` be more targeted toward the specific change in behavior?